pkgs/applications/misc/tilix/no-dub.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/meson.build b/meson.build
+index feff315..cb92631 100644
+--- a/meson.build
++++ b/meson.build
+@@ -93,12 +93,8 @@ source_root = meson.source_root()
+ sources_dir = include_directories('source/')
+ 
+ # Dependencies
+-# Note relying on dub to fetch and build dependencies before running meson build is temporary due to a bug in Meson 0.48
```

Comment:
I asked for clarification about the bug: 
https://github.com/gnunn1/tilix/commit/4092dfc7df6f8a2d2c90b29cf43995daac080a04#commitcomment-31853141

Reply:
Didn't particularly look around to see if there was an issue with this, but I did see improvements for dlang hit in 0.49.0.